lib/MDTopNav.js

```diff
@@ -121,7 +121,7 @@ var MDTopNav = function (_Component) {
   return MDTopNav;
 }(_react.Component);
 
-MDTopNav.propsTypes = {
+MDTopNav.propTypes = {
```

Comment:
@ZhengHe-MD good thing about linting is that it catches something you never expected